tpu/src/test/java/tpu/QueuedResourceIT.java

```diff
@@ -105,7 +98,10 @@ public void testGetQueuedResource() throws IOException {
   }
 
   @Test
-  public void testDeleteForceQueuedResource() {
+  public void testDeleteForceQueuedResource()
+          throws IOException, InterruptedException, ExecutionException {
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(bout));
```

Comment:
nit: why do you need this? it should be OK to print to stdout during the test unless you need to intercept this.

Reply:
The [validation](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/aa950a47cd5afab955cc91405ed37b4f21c37b59/tpu/src/test/java/tpu/QueuedResourceIT.java#L117) that you reference is not exact. Additionally, it is based on the non-functional status. Such validations result in transient errors.
I am sure we discussed the guidelines that require not using `stdout` in test assertions in the past.
I would appreciate it if you will be more attentive to the guidelines and there will be no need to repeat the same review feedback in multiple submissions.

Reply:
Thank you! Deleted verification